st2common/st2common/openapi.yaml.j2

```diff
@@ -4885,6 +4885,9 @@ definitions:
       secret:
         description: Encrypt value before saving the value.
         type: boolean
+      decrypt:
```

Comment:
This is a surprise.  How come we are adding decrypt attribute in this API spec?

Reply:
You are commenting on an old diff, attribute is now named ``pre_encrypted``.